src/assets/translations/en/main.json

```diff
@@ -1067,7 +1067,12 @@
       "insufficientValidTo": "Order expiry too soon",
       "excessiveValidTo": "Order expiry too far into the future",
       "insufficientLiquidity": "Insufficient liquidity",
-      "zeroFunds": "Non-zero sell asset balance needed"
+      "zeroFunds": "Non-zero sell asset balance needed",
+      "insufficientFundsForGas": "Insufficient funds for gas"
```

Comment:
Kind of related but also unrelated to this diff, nice-to-cleanup as followup: I was looking at those translations just this morning and we should probably dedupe them, this makes debugging pain and this addition will add one more addition
<img width="1239" alt="image" src="https://github.com/user-attachments/assets/9a7ee424-43d7-47fb-98cb-18b15f95f13f" />